fink_science/hostless_detection/processor.py

```diff
@@ -130,7 +130,8 @@ def run_potential_hostless(
         c4 = finkclass[index] in CONFIGS["finkclass"]
         c5 = tnsclass[index] in CONFIGS["tnsclass"]
         c6 = abs(deltat[index]) <= CONFIGS["cutout_timeframe"]
-        if ((c0[index] or c1[index] or c2[index] or c3[index] or c4 or c5) and c6):
+        c7 = sum(magpsf[index] <= CONFIGS["cutout_magnitude"]) > CONFIGS["minimum_number_of_alerts"]
```

Comment:
Just to understand what you want to do: as it is written, it is sufficient to have 1 point in the lightcurve below the mag threshold to process the event. `magpsf[index]` is an array of float (the magnitude for each alert of an object). So as long as there is 1 measurement in the lightcurve below `CONFIGS["cutout_magnitude"]`, your condition becomes `True`:

```python
print(sum([True, True, True, True])) # process lightcurve
4 (True)

print(sum([True, False, False, False])) # process lightcurve
1 (True)

print(sum([True, True, False])) # process lightcurve
2 (True)

print(sum([False, False, False])) # reject lightcurve
0 (False)
```

Is that what you have in mind? 

Or you want ALL measurements to be below the threshold, in which case you should use `np.all`:

```python
print(np.all([True, True, True, True])) # process lightcurve
True

print(np.all([True, False, False, False])) # reject lightcurve
False

print(np.all([True, True, False])) # reject lightcurve
False

print(np.all([False, False, False])) # reject lightcurve
False
```

Reply:
@JulienPeloton The former condition. There should be at least more than 3 measurements that are less than or equal to `CONFIGS["cutout_magnitude"]` to the process the candidate. Thanks for verifying

Reply:
oops you are write, I misread the line. All good then!